docs/configuration.md

```diff
@@ -205,7 +205,7 @@ custom_css:
   - /assets/css/custom.css
 ```
 
-NOTE: This approach is deprecated. It is recommended to use [this](./configuration.md#custom_css) approach instead.
+NOTE: This approach is deprecated. It is recommended to use [this](configuration.md#custom_css) approach instead.
```

Comment:
I feel "this" has the same effect as "here" in this link. Something more descriptive might be better. Also, I'm not sure where this should point to, it seems to circle back to the same section.

Reply:
Agree  that instances of 'this' should also be changes to more descriptive links, as is being done with 'here'.

Reply:
Great points. I just pushed up some updates. @SavvasStephanides @AnnCorp This is ready for review again.

Reply:
Happy with the changes here.